backup_and_restore/application_backup_and_restore/troubleshooting.adoc

```diff
@@ -0,0 +1,27 @@
+[id="troubleshooting"]
+= Troubleshooting
+include::modules/common-attributes.adoc[]
+:context: oadp-troubleshooting
+:oadp-troubleshooting:
+:namespace: openshift-adp
+:local-product: OADP
+:must-gather: registry.redhat.io/oadp-operator/oadp-must-gather:latest
```

Comment:
I don't know if that's the tag, currently I cannot download it from registry.redhat.io or from prestage.

Reply:
I don't know either. It probably is not ready either, because it was not in the build that was pushed recently. I'm just putting in whatever bits I have. We do know that it will be on registry.redhat.io and it will probably be called oadp-must-gather. The tag will probably be version-specific, because we don't usually want the user to automatically get the latest. We want them to get a 'real' release version. This is just a placeholder until I get confirmation from Dylan.